test-framework/src/main/java/org/keycloak/test/framework/util/UIUtils.java

```diff
@@ -0,0 +1,29 @@
+package org.keycloak.test.framework.util;
+import org.apache.commons.lang3.StringUtils;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+
+public final class UIUtils {
+
+    public static final String VALUE_ATTR_NAME = "value";
+
+    public static String getTextInputValue(WebElement input) {
+        return input.getAttribute(VALUE_ATTR_NAME);
+    }
+
+    public static void setTextInputValue(WebElement input, String value) {
+        input.click();
+        input.clear();
+        if (!StringUtils.isEmpty(value)) { // setting new input
+            input.sendKeys(value);
+        }
+        else { // just clearing the input; input.clear() may not fire all JS events so we need to let the page know that something's changed
```

Comment:
Is there still no better way to work around this? Seems like a hack :(

Reply:
Will remove whole class for now